components/authz-service/server/v2/policy_test.go

```diff
@@ -919,19 +1074,17 @@ func TestUpdatePolicy(t *testing.T) {
 		}},
 		{"successfully updates policy statements", func(t *testing.T) {
 			storedPol, _ := addSomePoliciesToStore(t, store, prng)
-			// change original statement effect to allow
```

Comment:
Comments in this test were outdated (read: wrong)... I repurposed the test to handle both explicit and implicit "all" cases.